lessons/Algebra2/exploring-linear-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,265 @@
+
+These models are just functions being graphed on top of the scatter plot, with the goal of minimizing the distance between the line and all the points on the plot. For straight-line relationships, these are _linear functions_ or "linear models". The straight-line graph of these models is sometimes called the "regression line" or the "predictor function", but you may have heard it called the @vocab{line of best fit}.
+
+When we make a model, we want it to be the closest possible approximation of all the points. A "good fit" has most of the points very close to the line, and a "bad fit" has the points very far away.
+
+You may already be familiar with the different forms of linear models:
+
+[cols="^1,^1,^1", options="header"]
+|===
+| Standard Form 	| Point-Slope Form			| Slope-Intercept Form
+| @math{Ax+By = C}	| @math{y-y_1 = m(x-x_1)}	| @math{y = mx+b}
+|===
+
+Depending on what part of the model we care about most, we might choose to use one display over another.
+
+- Standard Form makes it easy to find the x- and y-intercepts of the line
+- Point-Slope Form makes it easy to find the equation of the line given a single point and slope
+- Slope-Intercept Form makes it really easy to read off the slope and y-intercept
+
+Since we're looking for both the slope and the y-intercept for this model, it makes sense to use Slope-Intercept form.
+
+
+=== Investigate
+
+[.lesson-instruction]
+- Complete the first section ("Build a Model from Samples") in @printable-exercise{model-college-v-income.adoc}.
+- How well did your model work for Alabama and Alaska? Why didn't it work as well for other states?
+- How can we measure "how well a model fits"?
+
+Before students complete the remainder of the workbook page, confirm that they were able to successfully compute slope and y-intercept, define and test `f(x)` in Pyret, and evaluate the predictive value of `f(x)`.
+
+Pyret includes a function called `fit-model`. Find this contract in your contracts pages! Like `scatter-plot`, it consumes columns for our labels, our @math{x}s and ours @math{y}s. However, it __also consumes a function!__ It produces a scatter plot, with the function graphed on top of it.
+
+[.lesson-instruction]
+- Complete the second section ("Fit the Model") in @printable-exercise{model-college-v-income.adoc}.
+- What does @math{R^2} mean?
+
+@math{R^2} describes the _percentage of the variation in the y-variable that is explained by the x variable_ in our model. In other words, an @math{R^2} value of 0.20 could mean that “20% of the variation in median household income is explained by the percentage of college degrees in a state, according to our linear model”. Better models will explain a higher percentage of that variation!
```

Comment:
the glossary states:

> a number between 0 and 1 that measures the strength of association between two variables

should we add something that acknowledges that R^2 is measuring the efficacy of **our model** in predicting y?

Reply:
fixed